index.src.html

```diff
@@ -1841,73 +1849,269 @@
+        |init|.{{AudioDataInit/data}}.
+    8. Let |resourceReference| be a reference to |resource|.
+    9. Assign |resourceReference| to {{AudioData/[[resource reference]]}}.
+2. Return |frame|.
+
+### Attributes ###{#audiodata-attributes}
+
+: <dfn attribute for=AudioData>sampleFormat</dfn>
+:: The {{AudioSampleFormat}} used by this {{AudioData}}.
+
+    The {{AudioData/sampleFormat}} getter steps are to return
+    {{AudioData/[[sample format]]}}.
+
+: <dfn attribute for=AudioData>sampleRate</dfn>
+:: The sample-rate, in Hz, for this {{AudioData}}.
+
+    The {{AudioData/sampleRate}} getter steps are to return
+    {{AudioData/[[sample rate]]}}.
+
+: <dfn attribute for=AudioData>numberOfFrames</dfn>
+:: The number of frames (samples per channel) for this {{AudioData}}.
+
+    The {{AudioData/numberOfFrames}} getter steps are to return
+    {{AudioData/[[number of frames]]}}.
+
+: <dfn attribute for=AudioData>numberOfChannels</dfn>
+:: The number of audio channels for this {{AudioData}}.
+
+    The {{AudioData/numberOfChannels}} getter steps are to return
+    {{AudioData/[[number of channels]]}}.
+
+: <dfn attribute for=AudioData>allocationSize</dfn>
+:: The the number of bytes allocated to hold all of the samples in this
+    {{AudioData}}.
+
+    The {{AudioData/allocationSize}} getter steps are to:
+    1. Let |sampleSize| be the number of bytes per sample, as defined by the
+        {{AudioData/[[sample format]]}}.
+    2. Return the product of multiplying |sampleSize| by
+        {{AudioData/[[number of channels]]}} and
+        {{AudioData/[[number of frames]]}}.
+
+: <dfn attribute for=AudioData>timestamp</dfn>
+:: The presentation timestamp, in microseconds, for this {{AudioData}}.
+
+    The {{AudioData/numberOfChannels}} getter steps are to return
+    {{AudioData/[[timestamp]]}}.
+
+: <dfn attribute for=AudioData>duration</dfn>
+:: The duration, in microseconds, for this {{AudioData}}.
+
+    The {{AudioData/duration}} getter steps are to:
+    1. Let |microsecondsPerSecond| be `1,000,000`.
+    2. Let |durationInSeconds| be the result of dividing
+        {{AudioData/[[number of frames]]}} by {{AudioData/[[sample rate]]}}.
+    3. Return the product of |durationInSeconds| and |microsecondsPerSecond|.
+
+### Methods ###{#audiodata-methods}
+: <dfn method for=AudioData>
+      copyTo(|destination|, |planeNumber|)
+    </dfn>
+:: Copies the samples from the specified plane of the {{AudioData}} to the
+    destination buffer.
 
+    When invoked, run these steps:
+    1. If the value of |frame|'s {{AudioData/[[detached]]}} internal slot is
+        `true`, throw an {{InvalidStateError}} {{DOMException}}.
+    2. Let |allocationSize| be the number of bytes allocated to hold this
+        {{AudioData}}'s [=media resource=], as described by the getter steps of
+        {{AudioData/allocationSize}}.
+    3. If |allocationSize| is greater than `destination.byteLength`, throw a
+        {{TypeError}}.
```

Comment:
Isn't that usually `RangeError` [from ES](https://tc39.es/ecma262/#sec-native-error-types-used-in-this-standard-rangeerror) ? It would have been `IndexSizeError`, but that's deprecated.